test/compress/global_defs.js

```diff
@@ -123,6 +126,8 @@ mixed: {
         console.log(CONFIG);
     }
     expect: {
+        const FOO = { BAR: 0 };
+        console.log(FOO.BAR);
```

Comment:
This is a tricky case. But since `FOO` is known to global I think `FOO.BAR` should be changed to `0`.

Had `FOO` been a parameter, then it would have been left as is.